cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -274,7 +291,7 @@ importType sc env ty =
         C.TVBound v -> case Map.lookup (C.tpUnique v) (envT env) of
                          Just (t, j) -> incVars sc 0 j t
                          Nothing -> panic "importType TVBound" []
-    C.TUser _ _ t  -> go t
+    C.TUser _ _ t  -> go t -- ignore type synonym annotation.
```

Comment:
This is a minor quibble, but `TUser` isn't an annotation (in the sense that the `t` in `e : t` is an annotation). Rather, `TUser` is an identifier that expands to some other type. I'd prefer phrasing this as simply:

```suggestion
    C.TUser _ _ t  -> go t -- ignore type synonyms
```

Or perhaps:

```suggestion
    C.TUser _ _ t  -> go t -- look through type synonyms
```